ethicml/preprocessing/train_test_split.py

```diff
@@ -160,6 +160,46 @@ def __call__(
         return train_test_split(data, self.train_percentage, random_seed) + (split_info,)
 
 
+def generate_proportional_split_indexes(
+    data: DataTuple, train_percentage, random_seed: int = 42
+) -> Tuple["np.ndarray[np.int64]", "np.ndarray[np.int64]"]:
```

Comment:
add `from __future__ import annotations` at the top of the file and you won't need the quotes

Reply:
nevermind. Apparently we still support 3.6